src/main/kotlin/app/Config.kt

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app
+
+import app.utils.Options
+
+/**
+ * Config data class.
+ */
+data class Config (
```

Comment:
I think it can: protobuf structures can be built from file, written to file. Also there are setters defined on builders. So whats missed?